### ISSUE TITLE
Missing accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#54]
Hello! I noticed that the study group map is missing some accessibility features. It would be great if you could add them to make the map more inclusive and user-friendly for everyone. Thank you for your hard work!
